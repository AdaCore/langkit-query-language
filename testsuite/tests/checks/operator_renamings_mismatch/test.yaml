driver: 'checker'
rule_name: 'Operator_Renamings'
project: 'prj.gpr'
rule_arguments:
   operator_renamings.name_mismatch: 'true'
