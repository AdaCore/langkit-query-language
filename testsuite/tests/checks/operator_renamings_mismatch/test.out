renamings.ads:12:13: rule violation: renaming of an operator
12 |    function "+" (I, J : Integer)           --  FLAG
   |             ^^^

